sunpy/lightcurve/sources/stereo.py

```diff
@@ -0,0 +1,837 @@
+	>>> let.peek()
+	
+	References
+	----------
+	| http://www.srl.caltech.edu/STEREO/Level1/LET_public.html
+	
+	"""
+
+	# def peek(self, title="GOES Xray Flux"):
+	#     """Plots GOES light curve is the usual manner"""
+	#     figure = plt.figure()
+	#     axes = plt.gca()
+	
+	#     dates = matplotlib.dates.date2num(parse_time(self.data.index))
+	
+	#     axes.plot_date(dates, self.data['xrsa'], '-',
+	#                  label='0.5--4.0 $\AA$', color='blue', lw=2)
+	#     axes.plot_date(dates, self.data['xrsb'], '-',
+	#                  label='1.0--8.0 $\AA$', color='red', lw=2)
+	
+	#     axes.set_yscale("log")
+	#     axes.set_ylim(1e-9, 1e-2)
+	#     axes.set_title(title)
+	#     axes.set_ylabel('Watts m$^{-2}$')
+	#     axes.set_xlabel(datetime.datetime.isoformat(self.data.index[0])[0:10])
+	
+	#     ax2 = axes.twinx()
+	#     ax2.set_yscale("log")
+	#     ax2.set_ylim(1e-9, 1e-2)
+	#     ax2.set_yticks((1e-9, 1e-8, 1e-7, 1e-6, 1e-5, 1e-4, 1e-3, 1e-2))
+	#     ax2.set_yticklabels((' ', 'A', 'B', 'C', 'M', 'X', ' '))
+	
+	#     axes.yaxis.grid(True, 'major')
+	#     axes.xaxis.grid(False, 'major')
+	#     axes.legend()
+	
+	#     # @todo: display better tick labels for date range (e.g. 06/01 - 06/05)
+	#     formatter = matplotlib.dates.DateFormatter('%H:%M')
+	#     axes.xaxis.set_major_formatter(formatter)
+	
+	#     axes.fmt_xdata = matplotlib.dates.DateFormatter('%H:%M')
+	#     figure.autofmt_xdate()
+	#     figure.show()
+	
+	#     return figure
+
+
+	@staticmethod
+	def _parse_txt(filepath):
+		"""
+		Parses a STEREO LET file from
+		http://www.srl.caltech.edu/STEREO/Public/LET_public.html
+		
+		""" 
+		type_of_data = ''
+		header = []
+		data_read = {'data_type_read': False, 'header_line_reached': False }
+		
+		#To Read in header and get index of line where data starts
+		fp = open(filepath)
+		for i,line in enumerate(fp):
+			#To Find the type of data, 27 day, 1 day, 1 hour, 10 min or 1 min
+			if data_read['data_type_read'] == False:
+				if i == 2 and line[:3] == 'Bin':
+					type_of_data = '27day'
+					data_read['data_type_read'] = True
+				elif i in [5,6] and line[:4] == 'Time': 
+					type_of_data = line[17:].strip('\n')
+					data_read['data_type_read'] = True
+				else:
+					continue
+	
+		# To figure out and customise header in case of 27 day type of data
+		if type_of_data == '27day':
+		if line[:3] == 'Bin':
+			header = header + [line]
+			continue
+		elif line[:7] == 'Columns':
+			data_start = i + 4
+			data_points = int(line[20])
+			
+			for k in range(data_points):
+				header = header + ['Uncertainty for ' + header[k]]
+				header[k] = 'Flux for ' + header[k]
+			
+			header = ['Datetime'] + header
+			break
+	       
+		# To figure out and customise header in case of any other type of data since their files are formatted 
+		# similarly but different from 27 day type of data
+		else:
+		if data_read['header_line_reached'] == False and line[:6] != 'Column':
```

Comment:
should this `if` goes indented inside the `else` or `else` is empty? if empty, why to have it?